apps/cricketscoreapi/.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+          NEXT_PUBLIC_ENABLE_REAL_TIME: true
+          NEXT_PUBLIC_ENABLE_NOTIFICATIONS: false
+          # Clerk Authentication - Required for build process
+          NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY: ${{ secrets.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY }}
+          CLERK_SECRET_KEY: ${{ secrets.CLERK_SECRET_KEY }}
+
+  build-check:
+    name: ✅ Build Status
+    runs-on: ubuntu-latest
+    needs: [lint-and-build]
+    if: always()
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)